src/server/session.ts

```diff
@@ -2034,6 +2033,14 @@ export class Session<TMessage = string> implements EventSender {
         return this.mapTextChangesToCodeEdits(changes);
     }
 
+    private getCopilotRelatedInfo(args: protocol.FileRequestArgs): protocol.CopilotRelatedItems {
+        const { file, languageService } = this.getFileAndLanguageServiceForSyntacticOperation(args);

```

Comment:
This doesnt look like syntax operation : need to use `const { file, project } = this.getFileAndProject(args);`